lib/api/core/entrypoints/embedded/protocols/http.js

```diff
@@ -74,30 +73,32 @@ class HttpProtocol extends Protocol {
     this.decoders = this._setDecoders();
 
     this.server.on('request', (request, response) => {
+      const ips = [request.socket.remoteAddress];
+
+      if (request.headers['x-forwarded-for']) {
```

Comment:
Nitpicking:
If I understand well, the RFC allow another header to indicate the source ip https://tools.ietf.org/html/rfc7239#section-4  
eg: `Forwarded: for=192.0.0.2, for=192.0.0.3`

Reply:
Hmm.
It seems that it's the other way around: `Forwarded` is the standard header, and `x-forwarded-for` is a simplified version of it, not part in any official specification.
Since I lack expertise with the HTTP protocol: @ballinette, @benoitvidis > do you have an advice about that?

Reply:
I did not know about `Forwarded` header, and as far as I know, `x-forwarder-for` is more commonly used.
But as `Forwarded` is the specs, we should support it as well.